src/main.js

```diff
@@ -1,16 +1,25 @@
 // Create variables targetting the relevant DOM elements here :point_down:
+
 var getRandomCoverButton = document.querySelector(".random-cover-button")
 var currentImage = document.querySelector(".cover-image");
-/// i think follow this guideline for the rest of the classes
+var currentTitle = document.querySelector(".cover-title");
+var currentTag1 = document.querySelector(".tagline-1");
+var currentTag2 = document.querySelector(".tagline-2");
+
+var makeCoverButton = document.querySelector(".view form-view hidden");
```

Comment:
Good target!

Reply:
I think we have a good basis for moving forward with this function!